scout/constants/indexes.py

```diff
@@ -57,12 +57,12 @@
         IndexModel(
             # Filterish index
             [
-                ("case_id", ASCENDING),
                 ("variant_type", ASCENDING),
+                ("case_id", ASCENDING),
                 ("sub_category", ASCENDING),
                 ("hgnc_ids", ASCENDING),
             ],
-            name="case_id_variant_type_sub_category_hgnc_ids",
+            name="variant_type_case_id_sub_category_hgnc_ids",
```

Comment:
This is an old thing from a consultant meeting, making it a little bit easier on the query scheduler, and making the indices a little less redundant.